includes/forms/template.php

```diff
@@ -1375,39 +1375,36 @@ function give_get_register_fields( $form_id ) {
 
 			$class = ( 'registration' === $show_register_form ) ? 'form-row-wide' : 'form-row-first';
 			?>
-			<div id="give-create-account-wrap-<?php echo $form_id; ?>"
-				 class="form-row <?php echo esc_attr( $class ); ?> form-row-responsive">
+
+			<?php
+			/**
+			 * If Guest Checkout is enabled, display label and checkbox - unchecked.
+			 * If Guest Checkout it disabled, display hidden checkbox - checked.
+			 * @since 2.9.6
+			 */
+			?>
+			<div id="give-create-account-wrap-<?php echo $form_id; ?>" class="form-row <?php echo esc_attr( $class ); ?> form-row-responsive">
```

Comment:
@kjohnson Do not remove html element in favor of Multi step form template because same code also render legacy for template. Please check whether or not these change produce unexpected result on legacy form template.

Reply:
I revert these changes.